eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -87,11 +87,11 @@ jobs:
             Write-Host "##vso[task.setvariable variable=CodeCoverageFilePattern]$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**\coverage.cobertura.xml"
           }
         displayName: Set variable for the project list file and coverage directory
-      - pwsh: >-
+      - script: >-
           dotnet test eng/service.proj
-          --filter '(TestCategory!=Manually) & (TestCategory!=Live) & ($(AdditionalTestFilters))'
+          --filter "(TestCategory!=Manually) & (TestCategory!=Live) & ($(AdditionalTestFilters))"
```

Comment:
I assume the double quotes won't break here if AdditionalTestFilters is not defined, because it's no longer powershell?

Reply:
Yes based on being able to see the full command line now in the logs it looks correct to me.